README.md

```diff
@@ -0,0 +1,106 @@
+<p align="center"><a href="#"><img width=60% alt="" src="https://docs.dask.org/en/latest/_static/images/dask-horizontal-white.svg"></a>
```

Comment:
I use the light theme on GH, and couldn't see the white logo text. :/

Maybe, we can use just the [logo-mark](https://marketing.dask.org/en/latest/_images/dask-icon.svg) because we don't have a theme-neutral logo? Or, go with the [dark text](https://marketing.dask.org/en/latest/_images/dask-horizontal.svg) because I'd assume more folks use the light theme (I might be wrong here)

![Screenshot 2022-01-05 at 4 22 20 PM](https://user-images.githubusercontent.com/33131404/148206068-cff77816-80a3-497d-9800-ba0da8138c17.png)

Reply:
@jsignell Ok I will add all of the install sections under the `<details>` tap To ensure maintainability and in order not to make the README file too large
___________________________________
@pavithraes ,
you are right I am using the dark theme and there are a lot of users who use it so it will be better if I add just the logo mark